ddl/placement/utils_test.go

```diff
@@ -119,3 +119,131 @@ func (t *testUtilsSuite) TestObjectIDFromGroupID(c *C) {
 		}
 	}
 }
+
+func (t *testUtilsSuite) TestGetLeaderDCByBundle(c *C) {
+	testcases := []struct {
+		name       string
+		bundle     *Bundle
+		expectedDC string
+	}{
+		{
+			name: "only leader",
+			bundle: &Bundle{
+				ID: GroupID(1),
+				Rules: []*Rule{
+					{
+						ID:   "12",
+						Role: Leader,
+						LabelConstraints: []LabelConstraint{
+							{
+								Key:    "zone",
+								Op:     "in",
+								Values: []string{"bj"},
```

Comment:
Test these cases too:
- Key is not "zone"
- Op is not "in"
- len(Values) > 1

Reply:
updated.